calico/felix/endpoint.py

```diff
@@ -357,7 +368,8 @@ def _configure_interface(self):
                 ips.add(futils.net_to_ip(ip))
             devices.set_routes(self.ip_type, ips,
                                self._iface_name,
-                               self.endpoint["mac"])
+                               self.endpoint["mac"],
+                               reset_arp=mac_changed)
```

Comment:
This crashes Felix if it's an IPv6 ipset because the reset_arp flag isn't allowed for IPv6.

Please update the unit test to run in IPv6 as well.